chia/cmds/wallet_funcs.py

```diff
@@ -988,19 +993,35 @@ def print_nft_info(nft: NFTInfo, *, config: Dict[str, Any]) -> None:
 
 async def list_nfts(args: Dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
     wallet_id = args["wallet_id"]
+    start: Optional[int] = args.get("start")
+    num: Optional[int] = args.get("num")
     try:
         config = load_config(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
-        response = await wallet_client.list_nfts(wallet_id)
+        count_response: Dict[str, Any] = await wallet_client.count_nfts(wallet_id)
+        count: int = count_response["count"]
+        response = await wallet_client.list_nfts(wallet_id, pagination=(start, num))
         nft_list = response["nft_list"]
         if len(nft_list) > 0:
-            for n in nft_list:
+            for i, n in enumerate(nft_list):
                 nft = NFTInfo.from_json_dict(n)
-                print_nft_info(nft, config=config)
+                sequence_num = i + 1 + (start if start is not None else 0)
+                print_nft_info(nft, config=config, sequence_info=(sequence_num, count))
         else:
-            print(f"No NFTs found for wallet with id {wallet_id} on key {fingerprint}")
+            if count:
+                print(f"No NFTs found at offset {start} with length {num}")
```

Comment:
Let's check for `num` of 0 being the cause of no NFTs listed to provide clear feedback in such corner cases?